docs-ref-conceptual/eclipse/azure-toolkit-for-eclipse-create-hello-world-web-app.md

```diff
@@ -17,29 +18,52 @@ ms.topic: article
 ms.workload: web
 ---
 
-# Create a Hello World web app for Azure using Eclipse
+# Create a Hello World web app for Azure App Service using Eclipse
 
-This tutorial shows how to create and deploy a basic Hello World application to Azure as a web app by using the [Azure Toolkit for Eclipse].
+Using open sourced [Azure Toolkit for Eclipse](https://marketplace.eclipse.org/content/azure-toolkit-eclipse) plugin, create and deploy a basic Hello World application to Azure App Service as a web app can be done in a few minutes.
 
```

Comment:
a basic Hello World -> a Hello World

you can create and deploy a Java hello world application to Azure App Service as a web app in a few minutes. 
Or
Creating and deploying ... can be done.